Source/Core/Updater/UI.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <Windows.h>
+
+#include "Common/Flag.h"
+
+namespace UI
+{
+void Loop();
+void Stop();
+
+void SetDescription(const std::string& text);
+void SetMarquee(bool marquee);
+void ResetProgress();
+void SetProgress(int current, int total);
+void IncrementProgress(int amount);
+
+extern HWND label_handle;
```

Comment:
Do these really need to be extern globals? Like, can we just not?

Reply:
+1, and move them to an anonymous namespace.